app/Providers/Filament/AdminPanelProvider.php

```diff
@@ -67,7 +67,9 @@ public function panel(Panel $panel): Panel
             ])
             ->authMiddleware([
                 Authenticate::class,
-            ]);
+            ])
+            ->databaseNotifications()
+            ->databaseNotificationsPolling('10s');
```

Comment:
We should probably move this to do this configuration within the `notifications` module. See how we add `->discoverResources` in the other Modules like `case`.

We can do this in a different PR if you would prefer though.